.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,21 @@
+---
+name: Bug
+about: Submit a bug report
+labels: bug, new
+---
+
+<!--
+Thanks for taking the time to improve cloud-init!
+
+Please include the following in your bug report:
+
+1. Your cloud provider
+2. Any appropriate cloud-init configuration you can provide us with
+3. cloud-init logs:
+  Run `sudo cloud-init collect-logs`. Add `--include-userdata` if
+  there is no sensitive information in your user data.
+
+Include a minimal, reproducible example (<https://stackoverflow.com/help/minimal-reproducible-example>), if possible.
+-->
+
+# Bug report
```

Comment:
Since we have a construct of what we want in the bug report, let's prompt that in our visible template (not just comments) Maybe we can even omit the comments if the template content is good enough so people don't have to scroll past it and ignore it:
```
---
name: Bug
about: Submit a bug report
labels: bug, new
---

<!--
Thanks for taking the time to improve cloud-init!

Please include the following in your bug report with
a minimal, reproducible example (<https://stackoverflow.com/help/minimal-reproducible-example>), if possible.
-->

# Bug report

<!-- bug description explaining unmet expectation or use-case -->

## Steps to reproduce the problem
<!--Provide any applicable user-data, config, commandline or procedure to reproduce this problem -->

## Environment details
- Cloud-init version: 
- Operating System Distribution: 
- Cloud provider, platform or installer type: 

## cloud-init logs (or `sudo cloud-init collect-logs`)
<!--
Please provide either the applicable excerpt of /var/log/cloud-init.log representing the failure or attach cloud-init-logs.tar.gz obtained by sudo cloud-init collect-logs or sudo collect-logs --including-userdata
-->
```